cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition/config.jelly

```diff
@@ -25,10 +25,14 @@
 
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler">
   <f:entry title="${%Script}" field="script">
-    <f:textarea checkMethod="post"/>
+    <div id="workflow-editor-wrapper" style="display: none; position: relative">
```

Comment:
What is going to happen to someone using a screenreader? Cf. [JENKINS-23148](https://issues.jenkins-ci.org/browse/JENKINS-23148). Will that just ignore the `display` attribute?